content/en/seavoice/tts_panel/tutorial.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "TTS Tutorial"
+description: "Tutorial for the SeaVoice TTS customization panel."
+date: 2022-05-12T08:48:57+00:00
+lastmod: 2022-05-12T08:48:57+00:00
+draft: false
+images: []
+menu:
+  seax:
+    parent: "tts_panel"
+weight: 102
+toc: true
+---
+
+## Customization Panel
+-------------------
+
+The **SeaVoice TTS Customization Panel** is a user-friendly interface that gives you fine-grained control over:
+
+* pronunciation
+* format
+* pacing
+* emphasis
+
+Here we demonstrate how to navigate and utilize the SeaVoice TTS Customization Panel:
+
+   <iframe width="100%" height="40%" src="https://www.youtube.com/embed/rbjxJGUeQew" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="border-radius: 30px;"></iframe>
```

Comment:
the video iframe does not look good. I have not figured out a good way to configure it on both mobile and desktop... try `height="350px"`

Reply:
<img width="674" alt="Screen Shot 2022-05-13 at 10 55 50" src="https://user-images.githubusercontent.com/20276372/168341176-9497d7d4-fc22-4ed2-9370-b05bdd4eef73.png">